central/reports/common/report_formatter.go

```diff
@@ -82,7 +84,7 @@ type WatchedImagesResult struct {
 
 // Format takes in the results of vuln report query, converts to CSV and returns zipped CSV data and
 // a flag if the report is empty or not
-func Format(deployedImagesResults []DeployedImagesResult, watchedImagesResults []WatchedImagesResult) (*bytes.Buffer, bool, error) {
+func Format(deployedImagesResults []DeployedImagesResult, watchedImagesResults []WatchedImagesResult, configName string) (*bytes.Buffer, bool, error) {
```

Comment:
Add a bool `isV2Config` and set it to true calling from `report_gen_impl.go`. I think we should keep the naming of v1 reports the same as before.